src/VisualStudio/Core/Def/Telemetry/VisualStudioWorkspaceTelemetryService.cs

```diff
@@ -59,7 +60,7 @@ protected override void TelemetrySessionInitialized()
                     WellKnownServiceHubService.RemoteHost,
                     nameof(IRemoteHostService.InitializeTelemetrySession),
                     solution: null,
-                    new object?[] { client.ClientId, settings },
+                    new object?[] { Process.GetCurrentProcess().Id, settings },
```

Comment:
curious what a process ID is needed for.

Reply:
Just for logging.